### DISCUSSION TITLE
Use of Singleton to define route and twill navigation requires new parameter since laravel 9.42

### DISCUSSION THREAD

**Discussion by tlesseli** [area17/twill#2031] [Q&A] [answered]
After updating to laravel 9.42, (https://laravel-news.com/laravel-9-42-0)  I got errors on the route definition and the twill navigation. 
It seems that a route definition using singleton requires 2 parameters : the name of the singleton and the class. 

In the documentation of twill there is no reference to the 2nd parameter. (https://twill.io/docs/crud-modules/cli-generator.html#singleton).  I've replaced it by a module, but then of course, the twill navigation, that needs a named route doesn't work anymore. 

Basically, I need to know what my second parameter should be in the route definition, as indicated in laravel (https://laravel.com/docs/9.x/controllers#singleton-resource-controllers) 

Any advice?

**Comment by kallefrombosnia** [comment, marked as answer]
Check #1961

**Reply by tlesseli** [reply]
Thanks !
